Cargo.lock

```diff
@@ -2,71 +2,94 @@
 name = "cargo-watch"
 version = "2.1.3"
 dependencies = [
- "docopt 0.6.59 (registry+https://github.com/rust-lang/crates.io-index)",
- "docopt_macros 0.6.59 (registry+https://github.com/rust-lang/crates.io-index)",
- "notify 1.2.1 (registry+https://github.com/rust-lang/crates.io-index)",
- "regex 0.1.26 (registry+https://github.com/rust-lang/crates.io-index)",
- "rustc-serialize 0.3.12 (registry+https://github.com/rust-lang/crates.io-index)",
- "time 0.1.24 (registry+https://github.com/rust-lang/crates.io-index)",
+ "docopt 0.6.66 (registry+https://github.com/rust-lang/crates.io-index)",
+ "docopt_macros 0.6.66 (registry+https://github.com/rust-lang/crates.io-index)",
+ "notify 1.2.2 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
Notify is now v2.0.0, probably want to change that in the Cargo.toml. API hasn't changed (the breaking change is not API-related) so should compile fine.